lemur/common/validators.py

```diff
@@ -42,20 +42,20 @@ def private_key(key):
 
 def sensitive_domain(domain):
     """
-    Determines if domain has been marked as sensitive.
+    Determines if domain has been marked as sensitive or matches restricted patterns.
     :param domain:
     :return:
     """
+    if SensitiveDomainPermission().can():
+        # User has permission, no need to check anything
```

Comment:
We used to have something like this:

https://github.com/Netflix/lemur/blob/0.4.0/lemur/tests/conftest.py#L183

I had removed that fixture when I removed most of the permission logic from the services and into the views. This may be an edge case where this particular fixture is useful again.